lib/stylesheet/manager.rb

```diff
@@ -43,12 +43,21 @@ def self.color_scheme_cache_key(color_scheme, theme_id = nil)
   end
 
   def self.precompile_css
-    targets = %i[desktop mobile desktop_rtl mobile_rtl admin wizard]
+    targets = %i[desktop mobile admin wizard]
+    targets += %i[desktop_rtl mobile_rtl admin_rtl wizard_rtl]
```

Comment:
```suggestion
    targets = %i[
      desktop 
      mobile 
      admin 
      wizard 
      desktop_rtl 
      mobile_rtl 
      admin_rtl 
      wizard_rtl
    ]
```

I don't think appending the RTL items in the array is necessary when we can just declare the array with the RTL items when assigning the variable.